c10/xpu/XPUStream.cpp

```diff
@@ -244,6 +244,15 @@ XPUStream getCurrentXPUStream(DeviceIndex device) {
   return XPUStreamForId(device, current_streams[device]);
 }
 
+XPUStream getDefaultXPUStream(DeviceIndex device) {
+  initXPUStreamsOnce();
+  if (device == -1) {
+    device = current_device();
+  }
+  check_device(device);
+  return XPUStreamForId(device, makeStreamId(StreamIdType::NORMAL, 0));
+}
+
```

Comment:
@EikanWang @gujinghui @wizzniu Could you please help to reveiw the XPU part of this PR, thanks!

Reply:
I'm not familiar with XPU code, maybe @guangyey can give some advice.